faunadb-scala/src/test/scala/faunadb/ClientSpec.scala

```diff
@@ -2229,6 +2231,50 @@ class ClientSpec
     }
   }
 
+  it should "parallel testing should be executed without exceptions" in {
```

Comment:
could you name this test in a bit different way, it already starts with "it should"
so it would sound better as
```
it should "throw no exceptions when running 500 queries in parallel"
```